scripts/actions/spells/trust/tenzen_ii.lua

```diff
@@ -13,6 +13,16 @@ end
 
 spellObject.onMobSpawn = function(mob)
     xi.trust.message(mob, xi.trust.messageOffset.SPAWN)
```

Comment:
This trust has chat lines: https://ffxiclopedia.fandom.com/wiki/Trust:_Tenzen_II